### PR TITLE
Replace icons

### LINKED ISSUE #1: Change icons images
Change the ugly icons images.

Examples:
<div>
Screenshot 2023-02-25 at 09 22 15
Screenshot 2023-02-25 at 09 21 45
Screenshot 2023-02-25 at 09 22 01
</div>

### LINKED ISSUE #2: Change the icons font
Choose a modern icon font and use it in the same places where it is now used.

Cools fonts out there:
[Remix Icon](https://remixicon.com/)
[Themify](https://themify.me/themify-icons)
[Feather](https://feathericons.com/)
[Eva Icons](https://akveo.github.io/eva-icons/#/)

Examples:
<div>
Screenshot 2023-02-25 at 09 25 52
Screenshot 2023-02-25 at 09 26 05
Screenshot 2023-02-25 at 09 26 11
</div>

### PR DESCRIPTION
**Pull request by ttoino**
Closes #1
Closes #2 

Replaces most icons (I can't find them all) with ones provided by Remix Icons.

image

### CODE REVIEW

File: css/icons.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+div.info,
+div.alerta {
+    background-image: none;
+    display: flex;
+    align-items: center;
+    gap: .5em;
+    padding-left: .5em;
+}
```
Comment (thePeras) [inline css/icons.css:52]: Can you remove this?
This will be covered by #30

**Review by thePeras — CHANGES_REQUESTED** [review]
Amazing job! ⭐

Notice some small issues:
- "ler mais" buttons at the homepage added by #47 not showing up.
- On the "Agenda", the today day has a red border that shouldn't exist

File: css/icons.css (hunk 1 above)
Comment (ttoino) [reply css/icons.css]: Done!

**Comment by ttoino** [comment]
> * "ler mais" buttons at the homepage added by [Minor changes in homepage design and agenda style #47](https://github.com/NIAEFEUP/sigarra-extension/pull/47) not showing up.

Should now be fixed

> * On the "Agenda", the today day has a red border that shouldn't exist

Not sure why that is, it does not show up for me

**Review by thePeras — APPROVED** [review]
Awesome! ✅ 🛫 

A nice improvement would be blocking the request to the old icons images, so they wouldn't be initialized. Job of #36.

**Pull request merged**